### ISSUE TITLE
dist broken?

### ISSUE THREAD

**Issue by wayne-nlt** [bjoerge/debounce-promise#7]
Hi there,
I have just updated to the latest version of debounce-promise and the dist/index.js file seems to be broken.
Instead of require("debounce-promise") returning a function, it returns an object with the default key as the function, see below:

```
Wayne:tmp wayne$ npm install debounce-promise
debounce-promise@2.1.0 ../node_modules/debounce-promise
Wayne:tmp wayne$ node
> debounce=require("debounce-promise")
{ default: [Function: debounce] }
> debounce
{ default: [Function: debounce] }
> debounce()
TypeError: debounce is not a function
    at repl:1:1
    at REPLServer.defaultEval (repl.js:262:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:431:12)
    at emitOne (events.js:82:20)
    at REPLServer.emit (events.js:169:7)
    at REPLServer.Interface._onLine (readline.js:211:10)
    at REPLServer.Interface._line (readline.js:550:8)
    at REPLServer.Interface._ttyWrite (readline.js:827:14)
>
```

The docs don't reflect a change in the way it should be called, so I expect it must be a bug with the cleanup work you did recently.

Thanks,
Wayne

**Comment by bjoerge** [comment]
Aah, this is because babel got upgraded. Babel pre v6 transpiled `export default debounce` to `module.exports = debounce` (or equivalent), but now it is on the `default` key. A good reminder of why its a bad idea to transpile ES6 imports meant for `require()` consumption. I think the best way to fix it would be to just use `module.exports` in source files.

I'll fix it as soon as I find time for it. Thanks for the report!

**Comment by wayne-nlt** [comment]
I've committed a fix for this, but i'm not sure if it is what you want to do in the long run, there is some discussion here: http://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/

**Comment by bjoerge** [comment]
I think the only sane thing to do is to continue using `module.exports`, as it works with both `import` statements and `require()`. Fixed and released as of v2.1.1
